logstash-core/spec/api/lib/commands/jvm_spec.rb

```diff
@@ -12,6 +12,7 @@
     before(:each) do
       allow(agent).to receive(:node_name).and_return("foo")
       expect_any_instance_of(LogStash::Api::Service).to receive(:agent).and_return(agent)
+      allow(subject).to receive(:uptime).and_return(10)
```

Comment:
Its a double, not a real instance so in this case the number doesn't matter #4667 is better testing strategy since we actually starting a logstash instance under the hood with a config and test the api end points.